src/braket/aws/aws_quantum_task.py

```diff
@@ -83,6 +84,9 @@ def create(
                 For example, for D-Wave:
                 `{"providerLevelParameters": {"postprocessingType": "OPTIMIZATION"}}`
 
+            tags (Dict[str, str]): Tags, which are Key-Value pairs to add to this quantum task,
```

Comment:
I'd suggest making sure that this renders properly in Sphinx and adding ` around the example